components/src/dynamo/vllm/tests/test_vllm_worker_factory.py

```
@@ -596,6 +609,81 @@ async def test_passes_snapshot_engine(self, factory: WorkerFactory) -> None:
+    endpoint.connection_id.return_value = "worker-1"
+    endpoint.serve_endpoint = AsyncMock()
+    runtime = Mock()
+    runtime.endpoint.return_value = endpoint
+
+    engine_client = Mock()
+    vllm_config = Mock(model_config=Mock())
+    engine_tuple: EngineSetupResult = (
+        engine_client,
+        vllm_config,
+        Mock(),
+        "/tmp/prom",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/prom"

(S108)